diagnostic.md

```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    In the Application Router, you define the routes / URLs for the entire app. The
+    router matches the current URL to the routes you've defined.
+    In an Ember route you can customize specific routes and provide data.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.